src/Parallel/traversals/greedy_color.jl

```diff
@@ -1,4 +1,29 @@
-function random_greedy_color(g::AbstractGraph{T}, reps::Integer) where {T<:Integer}
+function random_greedy_color(
+    g::AbstractGraph{T}, reps::Integer; parallel=:distributed
+) where {T<:Integer}
+    return if parallel === :threads
+        threaded_random_greedy_color(g, reps)
+    elseif parallel === :distributed
+        distr_random_greedy_color(g, reps)
+    else
+        error(
+            "Unsupported parallel argument '$(repr(parallel))' (supported: ':threads' or ':distributed')",
+        )
+    end
+end
+
+function threaded_random_greedy_color(g::AbstractGraph{T}, reps::Integer) where {T<:Integer}
+    local_best = Any[nothing for _ in 1:reps]
```

Comment:
Is there a reason to use `Any[nothing for _ in 1:reps]` instead of
```julia
Vector{Graphs.Coloring{T}}(undef, reps)
```
or
```julia
Vector{Union{Nothing, Graphs.Coloring{T}}}(nothing, reps)
```
?

Reply:
Nope, I just wasn't sure if that type was always guaranteed

Let's use `Vector{Graphs.Coloring{T}}(undef, reps)`